analyzers/src/SonarAnalyzer.Common/Extensions/AttributeDataExtensions.cs

```diff
@@ -46,6 +46,18 @@ public static bool TryGetAttributeValue<T>(this AttributeData attribute, string
             }
         }
 
+        /// <summary>
+        /// Returns the <see cref="AttributeUsageAttribute.Inherited"/> setting for the attribute associated with <paramref name="attribute"/>.
+        /// The returned value is in line with the runtime behavior of <see cref="System.Reflection.MemberInfo.GetCustomAttributes(bool)"/>.
+        /// </summary>
```

Comment:
I would remove this comment. To break it down:

> Returns the see cref="AttributeUsageAttribute.Inherited"/ setting 

This is in the name

>for the attribute associated with paramref name="attribute"/

This is not useful, it's obvious from the arguments

> The returned value is in line with the runtime behavior of <ee cref="System.Reflection.MemberInfo.GetCustomAttributes(bool)"/.

This is not relevant for `HasAttributeUsageInherited` but for `GetAttributesWithInherited`

```suggestion
```